go/src/koding/kites/kloud/provider/softlayer/stack.go

```diff
@@ -117,19 +117,30 @@ func (s *Stack) injectBootstrap(b *Bootstrap, guest map[string]interface{}) erro
 // injectKiteKeys creates a kite key for each virtual guest, and sets up
 // the runtime injection of said key using cloud-init/user_data
 func (s *Stack) injectKiteKeys(guest map[string]interface{}, name string) error {
-	s.Builder.InterpolateField(guest, name, "user_data")
+	count := 1
+	if n, ok := guest["count"].(int); ok && n > 1 {
+		count = n
+	}
 
-	kiteKey, err := s.BuildKiteKey(name, s.Req.Username)
-	if err != nil {
-		return err
+	var labels []string
```

Comment:
naturally - there's a todo for this: https://github.com/koding/koding/issues/9127